cookiecutter.json

```diff
@@ -10,5 +10,6 @@
   "release_branch": "Name of the release branch. The production jobs (model training, batch inference) defined in this stack pull ML code from this branch. Default: `release`",
   "read_user_group": "Display name of the user group to give read permissions to the ML jobs, integration test job runs, and machine learning resources created for this project. A group with this name must exist in both the staging and prod workspaces. Default: `users`",
   "include_feature_store": ["no", "yes"],
+  "include_data_monitoring": ["yes", "no"],
```

Comment:
Do we want to flip this since by default they'd have to sign up for the private preview first?

Reply:
Removed the parameter.